### PR TITLE
added contribution bot

### PR DESCRIPTION
**Pull request by animesh-007**
Added all contribution bot #59

### CODE REVIEW

**Comment by shubhank-saxena** [comment]
Hello @animesh-007 ! This looks good to me. Can you send me the instructions on how this is going to work, ie, how is this going to add contributors in readme?
I had a different bot approach in my mind, but want to know how you're going to achieve this (it looks it is automated)

**Comment by animesh-007** [comment]
> Hello @animesh-007 ! This looks good to me. Can you send me the instructions on how this is going to work, ie, how is this going to add contributors in readme?
> I had a different bot approach in my mind, but want to know how you're going to achieve this (it looks it is automated)

https://allcontributors.org/docs/en/bot/overview I used this.

**Comment by shubhank-saxena** [comment]
Okay. So we are on the same bot! Why did you then make changes in package.json?

**Comment by animesh-007** [comment]
> Okay. So we are on the same bot! Why did you then make changes in package.json?

I actually followed their official doc. It was written over their😅

**Comment by shubhank-saxena** [comment]
Sorry couldn't find that one. Can you point me to package.json settings in the doc?

**Comment by animesh-007** [comment]
> Sorry couldn't find that one. Can you point me to package.json settings in the doc?

https://allcontributors.org/docs/en/cli/installation

**Comment by shubhank-saxena** [comment]
That's for CLI version. We are using the bot one. That's why the changes in package.json are needed to be removed.

**Comment by animesh-007** [comment]
> That's for CLI version. We are using the bot one. That's why the changes in package.json are needed to be removed.

So basically I should remove changes from package.json and rest all is fine?

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "projectName": "all-contributors-cli",
+  "projectOwner": "jfmengels",
```
Comment (shubhank-saxena) [inline .all-contributorsrc:3]: project owner in our case is DSC TIET

File: .all-contributorsrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "projectName": "all-contributors-cli",
```
Comment (shubhank-saxena) [inline .all-contributorsrc:2]: Project name is officialWebsite

File: README.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
 # officialWebsite
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```
Comment (shubhank-saxena) [inline README.md:2]: This tag should just come adjacent to slack one!

File: package.json [hunk 4]
```diff
@@ -30,7 +30,9 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
-    "eject": "react-app-rewired eject"
+    "eject": "react-app-rewired eject",
+    "contributors:add": "all-contributors add",
```
Comment (shubhank-saxena) [inline package.json:34]: Remove all the changes you made in this file!

Comment (shubhank-saxena) [inline package.json:34]: Remove all changes in this file!

**Pull request closed without merging**